tests/rakshctl/image/create_image_only_test.go

```diff
@@ -116,32 +153,32 @@ func TestWithDifferentArgs(t *testing.T) {
 	}{
 		{
 			"Positive test",
-			[]string{"--initrd", tc.initrd, "--vmlinux", tc.vmlinux, "--skip-app", "--symmKeyFile", tc.keyFile},
+			[]string{"--initrd", tc.initrd, "--vmlinux", tc.vmlinux, "--skip-app", "--symmKeyFile", tc.keyFile, "--nonceFile", tc.nonceFile},
 			"",
 		},
 		{
 			"Missing vmlinux argument",
-			[]string{"--initrd", tc.initrd, "--skip-app", "--symmKeyFile", tc.keyFile},
+			[]string{"--initrd", tc.initrd, "--skip-app", "--symmKeyFile", tc.keyFile, "--nonceFile", tc.nonceFile},
```

Comment:
can we add a negative scenario for missing `--nonceFile` option.?

Reply:
sure